### PR TITLE
# Basic project skeleton

### LINKED ISSUE #4: Provide basic skeleton for the project
Basic project structure - to be built upon further. May be non-functional at first.

### PR DESCRIPTION
**Pull request by dk1844**
- github templates
 - licence check / build CI/CD templates
 - SBT build/dependencies/plugins
 - started README.md
 - backend service @ spring boot with dummy example endpoint at `/api/test`:
 
login-gateway-test-controller2

All in all, heavily inspired by other AbsaOSS projects (Atum-service, Enceladus, ...)

 Closes #4.

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2018 ABSA Group Limited
```
Comment (Zejnilovic) [inline project/Dependencies.scala]: some have a wrong year hear

File: publish.sbt [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  Developer(
+    id    = "dk1844",
+    name  = "Daniel Kavan",
+    email = "daniel.kavan@absa.africa",
+    url   = url("https://github.com/dk1844")
+  )
+)
+
+// TODO create a site on github.io  - Issue #1
+//ThisBuild / homepage := Some(url("https://github.com/AbsaOSS/login-gw"))
```
Comment (Zejnilovic) [inline publish.sbt:36]: this could be uncommented

File: project/Dependencies.scala (hunk 1 above)
Comment (dk1844) [reply project/Dependencies.scala]: Fixed, thanks.

File: publish.sbt (hunk 2 above)
Comment (dk1844) [reply publish.sbt:36]: Are you sure? My thinking was that such a site must exists first and only then the homepage can be embedded into the publish metadata.

**Review by jakipatryk — APPROVED** [review]
LGTM, checked code, ran `service / Tomcat / start`, checked if `/api/test` endpoint works.

**Pull request merged**